__test__/useCases/GetCommitmentsAndExportFile/GetCommitments/index.unit.test.ts

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import {
+  getCommitmentsOffChain,
+  getCommitmentsOnChain,
+} from "../../../../useCases/GetCommitmentsAndExportFile/GetCommitments/index";
+
+import { commitments } from "../../../../__mocks__/useCases/ExportCommitments/commitments";
+
+jest.mock("axios");
+
+describe("Suit of tests for get commitmens from some endpoint", () => {
+  const mockedAxios = axios as unknown as jest.Mocked<typeof axios>;
```

Comment:
Do we need this? I think after mocking we can use the `axios` variable directly with no problem.